db/migrate/20211202105020_create_users.rb

```diff
@@ -4,7 +4,7 @@ def change
       t.string :name
```

Comment:
There are some double migrations in the database which are causing migration errors. Kindly try running` rails db: migrate` and remove all the duplicate migrations.